github/compliance/report.py

```diff
@@ -696,15 +696,22 @@ def process_result(
         known_issues = _all_issues(repository)
 
         if action == PROCESSING_ACTION.DISCARD:
+            latest_processing_date = None
+            if scan_result.has_latest_processing_date:
+                latest_processing_date = scan_result.latest_processing_date
             github.compliance.issue.close_issue_if_present(
                 scanned_element=scan_result.scanned_element,
                 issue_type=issue_type,
                 repository=repository,
                 known_issues=known_issues,
                 ctx_labels=ctx_labels,
+                latest_processing_date=latest_processing_date,
             )
 
-            element_name = github.compliance.issue.name_for_element(scan_result.scanned_element)
+            element_name = github.compliance.issue.name_for_element(
+                scanned_element=scan_result.scanned_element,
+                latest_processing_date=latest_processing_date,
+            )
```

Comment:
the latter `name_for_element` func might receive a different `latest_processing_date` as the former one (because the sprint assignment which can change the latest processing date is in between), resulting in a different returned name